src/components/Privacy/PolicyLanguages.tsx

```diff
@@ -0,0 +1,39 @@
+import { Fragment, ReactElement } from 'react'
+import styles from './PolicyLanguages.module.css'
+import {
+  useCookieMetadata,
+  UsePolicyMetadata,
+  usePrivacyMetadata
+} from '@hooks/usePolicyMetadata'
+import Link from 'next/link'
+
+export default function PolicyLanguages({
+  label,
+  policies
+}: {
+  label?: string
+  policies?: UsePolicyMetadata['policies']
+}): ReactElement {
+  /* const privacyMeta = usePrivacyMetadata()
+  const cookieMeta = useCookieMetadata()
+  const { policies } = isCookiePolicy ? cookieMeta : privacyMeta */
+  console.log('policies', policies)
+  const slug = policies?.[0].slug
+
+  return (
+    <div className={styles.langSelect}>
+      <span className={styles.langLabel}>{label || 'Language'}</span>
+      <div className={styles.langOptions}>
+        {policies.map((policy, i) => {
+          const slug = `${policies?.[0].slug}/${policy.policy}`
+          return (
+            <Fragment key={policy.policy}>
```

Comment:
suggestion: rename the policy attribute in policy to `policyName` or `policyLangTag`to prevent `policy.policy`